fx-src/jsm_prefix.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015 - Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Really, we're Apache licensing rather than MPL2.0? Did someone in legal, or someone who knows more than me (Gerv?) already comment on this?